acceptance/lib/puppet/acceptance/install_utils.rb

```diff
@@ -212,27 +184,18 @@ def install_repos_on(host, project, sha, repo_configs_dir)
             arch = $3
 
             # If this isn't outdated yet it will be by end of week (4/3/2015)
-            deb = fetch(
-              "http://apt.puppetlabs.com/",
-              "puppetlabs-release-%s.deb" % version,
-              platform_configs_dir
-            )
-
-            list = fetch(
-              "http://%s/%s/%s/repo_configs/deb/" % [tld, project, sha],
-              "pl-%s%s-%s.list" % [project, sha ? '-' + sha : '', version],
-              platform_configs_dir
-            )
-
-            repo_dir = fetch_remote_dir("http://%s/%s/%s/repos/apt/%s" % [tld, project, sha, version], platform_configs_dir)
+            deb_url = "http://apt.puppetlabs.com/puppetlabs-release-%s.deb" % version
+
+            list_url = "http://%s/%s/%s/repo_configs/deb/" % [tld, project, sha] + "pl-%s%s-%s.list" % [project, sha ? '-' + sha : '', version]
+
             repo_loc = "/root/#{project}"
 
             on host, "rm -rf #{repo_loc}"
             on host, "mkdir -p #{repo_loc}"
 
-            scp_to host, deb, repo_loc
-            scp_to host, list, repo_loc
-            scp_to host, repo_dir, repo_loc
+            repo_dir = fetch_remote_dir(host, "http://%s/%s/%s/repos/apt/%s" % [tld, project, sha, version], repo_loc)
```

Comment:
This line was fixed from the last commits; instead of downloading to `platform_configs_dir` (which resolved to `~/#{platform_configs_dir}` on the host) we just download the repo directly to `~/#{project}`.